src/DmxInput.cpp

```diff
@@ -116,4 +167,6 @@ void DmxInput::end()
 
     // Unclaim the sm
     pio_sm_unclaim(_pio, _sm);
+
+    free((void*)_buf);
```

Comment:
Is the call to `free` necessary when the buffer isn't heap allocated?

Reply:
No, good catch! I was experimenting with a malloc`d buffer and forgot to remove this. I will remove it.